### PR TITLE
Update Quality Declaration to reflect QL 2 (#160).

### PR DESCRIPTION
**Pull request by hidmic**
Precisely what the title says.

### CODE REVIEW

File: QUALITY_DECLARATION.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ This document is a declaration of software quality for the `class_loader` packag
 
 # class_loader Quality Declaration
 
-The package `class_loader` claims to be in the **Quality Level 4** category.
+The package `class_loader` claims to be in the **Quality Level 2** category.
 
 Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Quality Categories in REP-2004](https://www.ros.org/reps/rep-2004.html) of the ROS2 developer guide.
```
Comment (Blast545) [inline QUALITY_DECLARATION.md]:
```suggestion
Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Quality Categories in REP-2004](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#package-quality-categories) of the ROS2 developer guide.
```

**Comment by Blast545** [comment]
I'd add a couple of comments I found while reviewing the file, please address them, github interface didn't alllow me to comment directly in the file:

Update 2.i to: All changes will occur through a pull request, check the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#change-control-process) for additional information.


Update text in 3.i to link to [features](https://github.com/ros/class_loader/tree/ros2#usage) described in README.md file.

Update 3.ii to say "There is documentation for all of the public API using docblocks"

**Comment by Blast545** [comment]
And this should be merged after `rcpputils` is updated to Q2 as well.

File: QUALITY_DECLARATION.md (hunk 1 above)
Comment (hidmic) [reply QUALITY_DECLARATION.md]: Made a tweak. See https://github.com/ros/class_loader/pull/160/commits/873892b6db04d3e779cda31f18a6a238c4061cae.

**Review by Blast545 — APPROVED** [review]

**Comment by chapulina** [comment]
> And this should be merged after rcpputils is updated to Q2 as well.

Good call, I missed that when looking at the dependency graph

**Review by ahcorde — APPROVED** [review]

File: QUALITY_DECLARATION.md [hunk 2]
```diff
@@ -114,7 +114,7 @@ This includes:
 
 Changes are required to make a best effort to keep or increase coverage before being accepted, but decreases are allowed if properly justified and accepted by reviewers.
 
-Current coverage statistics can be viewed [here](https://ci.ros2.org/job/ci_linux_coverage/85/cobertura/src_ros_class_loader_include_class_loader/) and [here](https://ci.ros2.org/job/ci_linux_coverage/85/cobertura/src_ros_class_loader_include_class_loader/). This package does not yet meet the 95% coverage guideline, but it is currently above 90%.
+Current coverage statistics can be viewed [here](https://ci.ros2.org/job/ci_linux_coverage/lastSuccessfulBuild/cobertura/src_ros_class_loader_include_class_loader/). This package does not yet meet the 95% coverage guideline, but it is currently above 90%.
```
Comment (chapulina) [inline QUALITY_DECLARATION.md]: Could you add a link to the developer guide explaining how to calculate coverage? Such as

```
A description of how coverage statistics are summarized from this page, can be found in the ["ROS 2 Onboarding Guide"](https://index.ros.org/doc/ros2/Contributing/ROS-2-On-boarding-Guide/#note-on-coverage-runs).
```

See this example: https://github.com/ros2/rcutils/pull/260/files#diff-92440392597dfc3b7adb67bc4ed481b3R118

Comment (hidmic) [reply QUALITY_DECLARATION.md]: Done in https://github.com/ros/class_loader/pull/160/commits/7f6b7254cd794b29b4805e02be1dd391fac7e1e5.

**Review by chapulina — APPROVED** [review]

**Comment by hidmic** [comment]
Alright, may I ask anyone of the maintainers to merge this PR?

**Review by Blast545 — APPROVED** [review]

**Comment by hidmic** [comment]
@nuclearsandwich ping.

**Review by nuclearsandwich — APPROVED** [review]
I reviewed only that all of the changes in this QD appear factual. Not that the result correctly updates the quality level from four to two. I trust that the other reviewers who've been doing more on the quality front have done so.

**Comment by nuclearsandwich** [comment]
Thanks to the other reviewers. Together with my own review this is ready for merge.

**Pull request merged**